phpstan.neon

```diff
@@ -0,0 +1,13 @@
+includes:
+  - phpstan-baseline.neon
+parameters:
+  level: max
```

Comment:
```suggestion
  phpVersion: 80100
  level: max
```

I suggest to run phpstan always in PHP 8.1, no matter which PHP version is used. I would prefer this, is a mostly run that on my Linux box directly instead using docker.

Reply:
I made this a serparate commit so we can preserve why the descision was made